Documentation.app/Contents/MacOS/Command-Line.playground/Pages/Creating a Playground Book for iPad.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,42 @@
+// nef:begin:header
+/*
+ layout: docs
+ title: Creating a Playground Book for iPad
+ */
+// nef:end
+
+/*:
+ ## 📲 Creating a Playground Book for iPad
+ 
+ Swift Playgrounds is a revolutionary app that makes possible to write Swift code on an iPad. In the latest updates, Swift Playgrounds 3.x has added a new feature: [UserModules](https://developer.apple.com/documentation/swift_playgrounds/structuring_content_for_swift_playgrounds/using_modules_to_share_code_in_a_playground_book); it lets you include swift code and make it available across multiple chapters like modules.
```

Comment:
This may get old and we won't remember to update it, so a better, future-proof way of wording it:
```suggestion
 Swift Playgrounds is a revolutionary app that makes possible to write Swift code on an iPad. Starting from Swift Playgrounds 3.x, the app has added a new feature: [UserModules](https://developer.apple.com/documentation/swift_playgrounds/structuring_content_for_swift_playgrounds/using_modules_to_share_code_in_a_playground_book); it lets you include Swift code and make it available across multiple chapters, like modules.
```